.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+# add system-probe label to any changes done in network, security, eventmonitor or ebpf packages
+component/system-probe:
+  - pkg/network/** # npm and usm
+  - pkg/security/** # cws
+  - pkg/ebpf/** # ebpf manager
+  - pkg/eventmonitor/** # new event monitor component
+  - cmd/system-probe/** # system-probe executable
+  - pkg/process/monitor/** # process monitor used by USM
+  - tasks/system_probe.py # invoke tasks
+
```

Comment:
Please add:
```
pkg/collector/corechecks/ebpf/**
pkg/util/kernel/**
```

Also, do we want the CI/E2E-related files to be included here?

Reply:
added the proposed pkgs.
we can always add later more paths